gallery/package.json

```diff
@@ -14,9 +14,11 @@
     "webpack": "webpack --env production --mode=production",
     "webpack-dev-server": "webpack serve",
     "test": "npm-run-all check-install lint-tests build jest",
+    "test-dark": "npm-run-all check-install lint-tests build jest-dark",
     "ci-test": "npm-run-all check-install ci-lint-tests build jest",
     "check-install": "yarn install --check-files --frozen-lockfile",
     "jest": "jest --testTimeout=120000",
+    "jest-dark": "THEME=DARK jest --testTimeout=120000",
```

Comment:
I'd rather use something slightly more specific as the environment variable name, maybe `RSC_GALLERY_THEME`

Reply:
> I'd rather use something slightly more specific as the environment variable name, maybe `RSC_GALLERY_THEME`

Absolutely, I changed it here: [48b2b2b](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/48b2b2bd6d3120f10387e0f2e1cda2078519e9b7)